### PR TITLE
Comment on our handling of personal data

### PR DESCRIPTION
**Pull request by encukou**
AFAIK, all is fine. Let's just be as transparent as possible – which shouldn't be that hard, given that we operate in public.
We do stuff in the public interest (which the GDPR does mention – but if you search for anything related to GDPR, you'll be drowned in recommendations for companies that are *not* acting in public interest).

Let's get explicit consent from presenters from now on. I don't think we actually need it, but it doesn't hurt.
Also, I'm not a lawyer.

### CODE REVIEW

**Comment by encukou** [comment]
CI failure is unrelated; will fix.

File: pyvocz/templates/personal-info.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        Audiovideální záznam přednášejícího nebo jiných návštěvníků akce,
+    </li>
+    <li>
+        Kontaktní nebo jiné údaje o přednášejícím, ktreré jsou obsaženy
+        v prezentaci („slajdech“) nebo popisu přednášky,
+        kterou přednášející poskytl organizátorům.
+    </li>
+</ul>
+<p>
+    Tyto údaje zpracovávají i následující společnosti:
```
Comment (hroncok) [inline pyvocz/templates/personal-info.html]: @encukou Co Google? bit.ly/prpylight

Comment (encukou) [reply pyvocz/templates/personal-info.html]: Nebo srazy.info, a Meetup?
Vypsal jsem služby, kam dávám informace v rámci `pyvo.cz`. Nemyslím si že je v mých silách řešit takhle centrálně organizaci v jednotlivých městech; tohle by se IMO mělo napsat na bit.ly/prpylight zvlášť.

Comment (hroncok) [reply pyvocz/templates/personal-info.html]: Přidal jsme do prylight malou noticku, snad to bude stačit k prokázání dobré vůle.

**Pull request merged**